stable/mysql/values.yaml

```diff
@@ -170,6 +170,9 @@ ssl:
 ## Example: 'Australia/Sydney'
 # timezone:
 
+# To be added to the deployment
+deploymentLabels: {}
```

Comment:
document in the readme.md as well

Reply:
@cpanato It is mentioned in the configuration section. Do you think it should be added somewhere else?